src/Sulu/Bundle/SearchBundle/EventListener/ContentMapperSubscriber.php

```diff
@@ -69,7 +69,9 @@ public function onNodeSave(ContentNodeEvent $event)
     {
         $structure = $event->getStructure();
 
-        if ($structure->getNodeState() === Structure::STATE_PUBLISHED) {
+        // TODO: if the structure is the homepage, index it.
+        //       for some reason the content mapper never maps the state to published for a homepage
+        if ($structure->getPath() == '' || $structure->getNodeState() === Structure::STATE_PUBLISHED) {
```

Comment:
@wachterjohannes in the ContentMapper the homepage "state" (publishjed / test) is never changed on purpose, so it is always test. Do you know why? https://github.com/sulu-cmf/sulu/blob/feature/search-mapping/src/Sulu/Component/Content/Mapper/ContentMapper.php#L461